app/models/speaker.rb

```diff
@@ -9,8 +9,15 @@ class Speaker < ActiveRecord::Base
 
```

Comment:
Though not part of this PR, I noticed this:  has_many :program_sessions, through: :user
If that is not used anywhere, let's remove it.  If it is used, I'd like to know where.  Pretty much a program session is only consumed by the program and schedule parts of the app.  No where would a user sign in and see their program_sessions.


has_many :speakers, through: :proposals